editor/editor_vcs_interface.cpp

```diff
@@ -237,31 +183,43 @@ void EditorVCSInterface::commit(String p_msg) {
 	}
 }
 
-Array EditorVCSInterface::get_file_diff(String p_identifier, TreeArea p_area) {
+List<EditorVCSInterface::DiffFile> EditorVCSInterface::get_file_diff(String p_identifier, TreeArea p_area) {
+	List<DiffFile> diff_files = List<DiffFile>();
 
 	if (is_addon_ready()) {
 
-		return call("_get_file_diff", p_identifier, p_area);
+		Array result = call("_get_file_diff", p_identifier, p_area);
+		for (int i = 0; i < result.size(); i++) {
+			diff_files.push_back(convert_diff_file(result[i]));
+		}
 	}
-	return Array();
-}
 
-Array EditorVCSInterface::get_previous_commits() {
+	return diff_files;
+}
 
+List<EditorVCSInterface::Commit> EditorVCSInterface::get_previous_commits() {
+	List<EditorVCSInterface::Commit> commits = List<EditorVCSInterface::Commit>();
```

Comment:
The `= List<EditorVCSInterface::Commit>()` can be omitted if you want, although not very important